modules/Lending.py

```diff
@@ -171,6 +173,9 @@ def lend_cur(active_cur, total_lended, lending_balances):
 
     loans = api.return_loan_orders(active_cur, loanOrdersRequestLimit[active_cur])
     loans_length = len(loans['offers'])
+    if hide_coins and Decimal(loans['offers'][0]['rate']) < Decimal(cur_min_daily_rate):
```

Comment:
loans['offers'][0] might throw an exception if no offers are returned from server (i.e. offers list is empty, which I saw happen during a spike in loan demands on XMR)